server/src/instant/db/model/triple.clj

```diff
@@ -708,8 +709,20 @@
       (tracer/add-data! {:attributes {:total-count @row-count}})
       {:row-count @row-count})))
 
+(defn iso8601-date-str->instant [x]
+  (if (string/includes? x "T")
+    (Instant/parse x)
```

Comment:
We might want to do `(.toInstant (ZonedDateTime/parse x))` because `Instant/parse` only supports UTC time zones. So `(.toInstant (ZonedDateTime/parse "2025-01-02T00:00:00-08"))` works but `(Instant/parse "2025-01-02T00:00:00-08")` doesn't.